lib/python/treadmill_aws/hostmanager.py

```diff
@@ -18,7 +18,7 @@ def render_manifest(key_value_pairs):
     """ Stub function to supply instance user_data during testing. """
 
     return "#cloud-config\n" + yaml.dump(
-        key_value_pairs, default_flow_style=False)
+        key_value_pairs, default_flow_style=False, default_style='\'')
```

Comment:
care to have a test? is it related to the otp problem?

Reply:
Added test. 
Yes, this is to deal with the issue of special characters in the OTP string- I've noticed a domain join failure rate around 5% without this in place.